spec/rails_helper.rb

```diff
@@ -6,7 +6,8 @@
 
 require "rspec/rails"
 require "shoulda/matchers"
-require "capybara/poltergeist"
+#require "capybara/poltergeist"
+require 'capybara-webkit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.